packages/tsconfig_base.json

```diff
@@ -22,8 +26,8 @@
       "es2017",
       "es2018",
       "esnext.asynciterable"
-    ]
+    ],
+    "typeRoots": ["../node_modules/@types"]
   },
-  "include": ["./**/*.ts", "./**/*.tsx"],
-  "exclude": ["./node_modules"]
+  "include": ["../config/typescript/**/*.d.ts"]
```

Comment:
Does it make sense to have an include here?  I thought all projects should specify their own includes?

Reply:
This should be removed